FilecoinFoundationWeb/filecoin-foundation
# FIX: duplicate id

## Changes

- We had an error for duplicate id on the `SearchInput` form - [Ticket](https://www.notion.so/filecoin/FF-V4-Fix-Console-Errors-ffa24e8ecada4a258a8caa5dd77e1861?pvs=4)
- included `id` prop in `SearchInput` to differentiate between mobile and web

## How to test
- test the search bar and check if you get an error (screenshot attached in the  [Ticket](https://www.notion.so/filecoin/FF-V4-Fix-Console-Errors-ffa24e8ecada4a258a8caa5dd77e1861?pvs=4))
